examples/loads-windowcontext-creative.html

```diff
@@ -0,0 +1,35 @@
+<!doctype html>
+<html ⚡>
+  <head>
+        <meta charset="utf-8">
+    <title>window.context Example</title>
+    <link rel="canonical" href="./regular-html-version.html">
+    <meta name="viewport" content="width=device-width,minimum-scale=1,initial-scale=1">
+    <style amp-boilerplate>body{-webkit-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-moz-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-ms-animation:-amp-start 8s steps(1,end) 0s 1 normal both;animation:-amp-start 8s steps(1,end) 0s 1 normal both}@-webkit-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-moz-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-ms-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-o-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}</style><noscript><style amp-boilerplate>body{-webkit-animation:none;-moz-animation:none;-ms-animation:none;animation:none}</style></noscript>
+    <script async src="https://cdn.ampproject.org/v0.js"></script>
+  </head>
+  <body>
+    <h1 style="font-size: 1em;"> window.context example </h1>
+    <p> Creative not loading? Make sure you put .max in the url for the page </p>
+
+    <h2> A4A + JS Creative </h2>
+    <amp-ad width="300" height="400"
+            type="fake"
+            data-use-a4a="true"
+            src="http://localhost:8000/examples/ampcontext-creative.html">
+    </amp-ad>
+    <div>
+
```

Comment:
if all you want is space, you can do it via CSS. not sure if empty lines here would work